VideoLocker/res/layout/fragment_add_response.xml

```diff
@@ -21,6 +21,8 @@
             android:paddingRight="@dimen/edx_margin"
             android:paddingTop="@dimen/widget_margin">
 
+            <include layout="@layout/discussion_user_profile_row" />
```

Comment:
Note that the `fragment_add_response` layout is almost identical to the `fragment_add_comment` layout, the only real difference being the difference in the strings ('response' vs 'comment'), so you could probably `<include/>` most of it from one shared layout, and make the rest of it (the views with the variable string texts) into stubs with all the other properties exported to a shared style instead.